frontend/src/locales/en-US.json

```diff
@@ -713,9 +713,9 @@
       "tip": "The policy is not applied retroactively.",
       "info": "For updating schema on the existing database, this setting controls whether the task requires manual approval.",
       "manual": "Require manual approval",
-      "manual-info": "Pending schema migration task will only be executed after it's manually approved.",
+      "manual-info": "Tasks will only be executed after being manually approved.",
       "auto": "Skip manual approval",
-      "auto-info": "Pending schema migration task will be executed automatically."
+      "auto-info": "Tasks will be approved and executed automatically if all task checks pass."
```

Comment:
Tasks will skip approval and be executed ...

Reply:
I think it’s not quite a “skip” because, by skip, it implies that there is no PendingApproval stage at all, and the task starts at the Pending stage. IMO "automatically approved" is more accurate here because that's what we do; we transit tasks into Pending if all task checks pass.